Content.Shared/Access/AccessLevelPrototype.cs

```diff
@@ -18,7 +18,7 @@ public sealed class AccessLevelPrototype : IPrototype
         [DataField("name")]
         public string Name
         {
-            get => _name ?? ID;
+            get => (_name is not null) ? Loc.GetString(_name) : ID;
             private set => _name = Loc.GetString(value);
```

Comment:
Won't this double set the localised name?

Reply:
> Won't this double set the localised name?

my bad. fixed